code/Model/SiteTree.php

```diff
@@ -1693,7 +1703,7 @@ protected function onBeforeWrite()
         }
 
         // Check to see if we've only altered fields that shouldn't affect versioning
-        $fieldsIgnoredByVersioning = ['HasBrokenLink', 'Status', 'HasBrokenFile', 'ToDo', 'VersionID', 'SaveCount'];
```

Comment:
`HasBrokenFile` and `HasBrokenLink` are the only two fields that actually exist anymore.

I thought about changing `VersionID` to `Version` (which is what the field is now named), but that would be a behavioural change. It would also be a bit weird: changing the version number in the `SiteTree` table without writing a version would mean that `SiteTree` and `SiteTree_Versions` could end up out of sync.